go/genesis/grpc.go

```diff
@@ -72,18 +74,26 @@ func (s *grpcServer) ToGenesis(ctx context.Context, req *pb.GenesisRequest) (*pb
 	if err != nil {
 		return nil, err
 	}
+	schedulerGenesis, err := s.schedulerBackend.ToGenesis(ctx, height)
+	if err != nil {
+		return nil, err
+	}
 
 	doc := api.Document{
 		// XXX: Tendermint doesn't support restoring from non-0 height.
 		// https://github.com/tendermint/tendermint/issues/2543
-		Height:     0,
-		ChainID:    genesisDoc.ChainID,
-		Time:       time.Now(),
-		EpochTime:  *epochtimeGenesis,
-		Registry:   *registryGenesis,
-		RootHash:   *roothashGenesis,
-		Staking:    *stakingGenesis,
-		KeyManager: *keymanagerGenesis,
+		Height:           0,
+		ChainID:          genesisDoc.ChainID,
+		ConsensusBackend: genesisDoc.ConsensusBackend,
+		Time:             time.Now(),
```

Comment:
Likewise unrelated to your PR, I would want dumps taken on different hosts to be byte for byte identical, this should probably use the block timestamp.